src/javaScript/games.json

```diff
@@ -4,7 +4,7 @@
       {
         "name": "The Witcher 3: Wild Hunt",
         "price": "40",
-        "img-cover": "the-witcher-3.png",
+        "imgCover": "the-witcher-3.png",
```

Comment:
Why the change? It is not consistent with the other key names

Reply:
i changed all the key names in the end due to the fact that prettier adds a space between the `-` so when i try to use it come out like this `${game.img - cover}` and JS doesn't read the spaces in between